packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.Advanced.Example.tsx

```diff
@@ -57,6 +88,8 @@ export class DetailsListAdvancedExample extends React.Component<{}, IDetailsList
       _items = createListItems(ITEMS_COUNT);
     }
 
+    this._getCommandItems = memoizeFunction(this._getCommandItems);
```

Comment:
Yes, it was...it was always being called with no parameters, and therefore always returning the same result. Fix is to pass in the values from state instead of accessing them directly.